src/ensembl/src/content/app/browser/track-panel/trackPanelState.ts

```diff
@@ -1,18 +1,31 @@
 import { TrackType } from './trackPanelConfig';
 import browserStorageService from '../browser-storage-service';
 
-const selectedBrowserTab = browserStorageService.getSelectedBrowserTab();
-
-export type TrackPanelState = Readonly<{
+export type TrackPanelStateForGenome = Readonly<{
   isTrackPanelModalOpened: boolean;
   isTrackPanelOpened: boolean;
-  selectedBrowserTab: { [genomeId: string]: TrackType };
+  selectedBrowserTab: TrackType;
   trackPanelModalView: string;
 }>;
 
-export const defaultTrackPanelState: TrackPanelState = {
-  isTrackPanelModalOpened: false,
-  isTrackPanelOpened: true,
-  selectedBrowserTab,
-  trackPanelModalView: ''
-};
+export type TrackPanelState = Readonly<{
+  [genomeId: string]: TrackPanelStateForGenome;
+}>;
+
+const selectedBrowserTabFromStorage = browserStorageService.getSelectedBrowserTab();
+
+// create the default state using the stored selected browser tab object
```

Comment:
this comment is redundant — it follows from variable names that the defaultTrackPanelState is created and it somehow uses selectedBrowserTabFromStorage for its creation.